generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
Well, the check in test_setup.py only works when there is "expected_hugepage_size" in cfg. 
If we don't set "expected_hugepage_size",  I don't think there is a way to check if the hugepage size is what we want. 
BTW, I tested boot_vm_in_hugepage.1G_hugepage on host whose hugepage size is 2M. Before the test, I removed the "expected_hugepage_size" line, and the case passed. Here comes the issue I mentioned above. Thanks!